src/components/errorbars/compute_error.js

```diff
@@ -30,18 +30,26 @@ module.exports = function makeComputeError(opts) {
         symmetric = opts.symmetric;
 
     if(type === 'data') {
-        var array = opts.array,
-            arrayminus = opts.arrayminus;
+        var array = opts.array || [];
 
-        if(symmetric || arrayminus === undefined) {
```

Comment:
Note that this (removing `|| arrayminus === undefined`) isn't actually a change in behavior, since we were filling in `[]` at the `supplyDefaults` step anyway